### DISCUSSION TITLE
How should one cite the OpenAPI specification?

### DISCUSSION THREAD

**Discussion by rpgoldman** [OAI/OpenAPI-Specification#2705] [Q&A]
Is there an authoritative citation form we should use (Bibtex, EndNote, etc.)?  If not, any chance someone involved would make one?

I looked at Google Scholar, DBLP, in the specification itself, and didn't find anything.

I can make my own, but probably wouldn't do as good a job as the OAI would.

**Comment by hkosova** [comment]
GitHub repositories support CITATION files by the way.

[About CITATION files](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-citation-files#about-citation-files)
[Citation File Format](https://citation-file-format.github.io/)

**Reply by rpgoldman** [reply]
I have tried to make a citation file for an article associated with one of my GitHub projects, and found that the GitHub citation fields wasn't fully ready for service yet.  Making a citation available in one of the standard formats (Bibtex, EndNote, etc.) would be a better solution, at least in the short term.

(The invention of the CFF format, IMO, was an unnecessary bit of NIH-age.  There are plenty of widely accepted formats for citations, and it would have been better to adopt one rather than develop a new one, which would require a new toolchain to support it.)

**Reply by earth2marsh** [reply]
I suspect we have a Venn diagram with a very narrow overlap (at best) between citation expertise and OpenAPI expertise! If you're willing to create a an example to start from, we can try to drum up help to improve it. Since I don't know what a good citation looks like, I'm at a loss to assist, myself.

**Comment by handrews** [comment]
No response to @earth2marsh 's request for a starting point since late 2021, closing.

**Comment by adelavega** [comment]
Is there still no formal citation for the OpenAPI spec? A DOI we can cite?
